linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/rule.yml

```diff
@@ -79,3 +72,13 @@ ocil: |-
     To check if the system login banner is compliant,
     run the following command:
     <pre>$ cat /etc/issue</pre>
+
+warnings:
+    - general: |-
+        The DoD "Policy on Use of Department of Defense (DoD) Information
+        Systems - Standard Consent Banner and User Agreement" requires
+        an exact text match, including line breaks and spaces, of the
```

Comment:
@shawndwells The regular expression used for this rule, defined in https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-banners/login_banner_text.var, allows arbitrary line breaks and spaces. Should https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-banners/login_banner_text.var be changed to require only exact text match?